kahuna/public/js/search/query.js

```diff
@@ -214,10 +218,10 @@ query.controller('SearchQueryCtrl', [
         if (isNonFree === null) {
           ctrl.filter.nonFree = session.user.permissions.showPaid ?
             session.user.permissions.showPaid : undefined;
-            storage.setJs("isNonFree", session.user.permissions.showPaid ? session.user.permissions.showPaid : false);
+            storage.setJs("isNonFree", session.user.permissions.showPaid ? session.user.permissions.showPaid : false, true);
```

Comment:
IIUC this will store this value in session storage, not local storage – is this desired? cc. @paperboyo

Reply:
(Ee think this might simply addressing an inconsistency with storage elsewhere? Good to confirm if so.)

Reply:
I wouldn’t know where we store current things we store? (panel locks etc). If there is no reason for divergence, I would store all things in the same place? But otherwise I have no opinion.

Reply:
Ace, @andrew-nowak suspect we're good to approve in this case?

Reply:
Thanks, @jonathonherbert and @paperboyo for the review,
 in the past, it was stored in local storage (browser scoped) and it led to unwanted behavior so it been stored in session storage (tab scoped) and for this specific change in line 221, it was missed in the previous [PR](https://github.com/guardian/grid/pull/3645)
as you can see the same in line [154](https://github.com/guardian/grid/blob/d9ab813eb1dd50513e320a87a173867efc44eb46/kahuna/public/js/search/query.js#L154) and line [176](https://github.com/guardian/grid/blob/d9ab813eb1dd50513e320a87a173867efc44eb46/kahuna/public/js/search/query.js#L176)